### DISCUSSION TITLE
Workspace can span multiple monitors

### DISCUSSION THREAD

**Discussion by drewzh** [nikitabobko/AeroSpace#1000] [questions-and-answers] [answered]
I have an admittedly fringe-case scenario whereby my main screen (Samsung Odyssey G9 59") displays 2x 4k video feeds (HDMI/DVI) side by side from the same Macbook using the monitors PBP mode. I've configured MacOS to treat them as one by disabling spaces, which allows me to centre position windows. The reason being this makes it possible for an M1 Pro Macbook to display a 120Hz desktop at full 7680x2160 resolution - something that isn't possible with a single cable with the M1's.

My question is this - before I found Aerospace, I used Apple Scripts and a Stream Deck to provide me with some rudimentary 'tiling' features, like displaying a window 1/3 width in the centre of the virtual screen (so basically 1/6 on one screen and the other 1/6 on the other).

Is it possible with AeroSpace to display a workspace than spans multiple screens? I don't necessarily need multiple workspaces - but that would be nice.

Apologies in advance if this isn't the right place to make queries - I didn't see an alternative channel.

**Comment by nikitabobko** [comment, marked as answer]
> Is it possible with AeroSpace to display a workspace than spans multiple screens? I don't necessarily need multiple workspaces - but that would be nice.

No, it's not possible, and it's unlikely will be possible, the priority of this feature is very minor
